cmd/gmailctl/cmd/apply_cmd.go

```diff
@@ -10,11 +10,12 @@ import (
 )
 
 var (
-	applyFilename     string
-	applyYes          bool
-	applyRemoveLabels bool
-	applySkipTests    bool
-	applyDebug        bool
+	applyFilename         string
+	applyYes              bool
+	applyRemoveLabels     bool
+	applySkipTests        bool
+	applyDebug            bool
+	applyDiffContextLines int
```

Comment:
Do you mind renaming this into `applyDiffContext` so it's named the same as the flag itself `--diff-context`?